### DISCUSSION TITLE
release schedule for v3.1.2?

### DISCUSSION THREAD

**Discussion by Bobgy** [argoproj/argo-workflows#6329] [Q&A] [answered]
I'm curious whether there's a fixed schedule or everything is ad hoc.
Where can I find latest ETA and progress towards a certain release?

context: I'm releasing Kubeflow Pipelines which depends on argo, a heads up help me decide which of the following I should choose:
* hold the upgrade (until bugs we found are fixed and released)
* fork & cherry pick the bug fixes we care about
* wait for the next patch release (and what will be included?)

**Comment by sarabala1979** [comment, marked as answer]
@Bobgy For Major GA releases, you can find the release date in milestones like 3.1, 3.2. Patch releases, It used to release every week if we have more issues on that release. Once the next GA version is released, the frequency of patch release will be reduced.  Currently, we are under stuff.  I will create cherry-pick list for 3.1.2 today. @hblixt and I are working to provide a roadmap and release days

**Reply by Bobgy** [reply]
Thank you so much!
